README.md

```diff
@@ -6,56 +6,57 @@
 
 ![logo](https://raw.githubusercontent.com/opentok/accelerator-core/master/tokbox-logo.png)
 
-# OpenTok Accelerator Core
+# OpenTok Accelerator Core Web
 
+OpenTok's `Accelerator Core Web` provides an easy integration for OpenTok libraries. Wether you've already built your application or are starting to use OpenTok's libraries, `Accelerator Core Web` helps you implement audio/video communication into your application with rich WebRTC features:
 
-```javascript
-otCore.init(options) =>
-  Multi-Party Audio/Video +
-  Text Chat +
-  Screen Sharing +
-  Annotation +
-  Archiving
-```
+  - Multi-Party Audio/Video Communication
+  - Text Chat
+  - Screen Sharing
+  - Annotation
+  - Archiving
+
+##Quickstart
 
-OpenTok Accelerator Core provides a simple way to integrate real-time audio/video into your web application using the OpenTok Platform.  `Core` also integrates with, manages, and provides a single API for the following accelerator packs:
+ To start using audio/video communication with OpenTok's libraries go to [Using accelerator core](#using-accelerator-core). To implement a more granular and complex management for OpenTok's session you can check [Advanced OpenTok Session Management](#advanced-opentok-session-management). 
+ 
+##Using accelerator core
+
+`OpenTok Accelerator Core` provides a simple way to integrate real-time audio/video into your web application using the OpenTok Platform.  It also integrates with, manages, and provides a single API for the following accelerator packs:
 
  - [Text Chat](https://www.npmjs.com/package/opentok-text-chat)
  - [Screen Sharing](https://www.npmjs.com/package/opentok-screen-sharing)
  - [Annotation](https://www.npmjs.com/package/opentok-annotation)
  - [Archiving](https://www.npmjs.com/package/opentok-archiving)
 
-`Core` also internally maintains the state of your OpenTok session for you.  Calling `otCore.state()` returns an object containing:
-```javascript
-  streams     => All current streams
-  streamMap   => The map of stream ids to publisher/subscriber ids
-  publishers  => All current publishers
-  subscribers => All current subscribers
-  meta        => The count of all current publishers and subscribers by type
-```
-*See an example of the publisher, subscriber, and meta data [below](#pubSubData).*
 
-Sample Applications
--------------------
+###Advanced OpenTok Session Management
+Even though `OpenTok Accelerator Core` will provide the necesary session management functionality for most projects, there might be cases where a more fine-grained control over the state of sessions in your application is required. For such cases you can use [`OpenTok JS SDK Wrapper`](./sdkWrapper.MD) which provides built-in state management. However, it **lacks management of the accelerator packs** commented [above](#using-accelerator-core). Consider using [`OpenTok JS SDK Wrapper`](./sdkWrapper.MD) if you must:
```

Comment:
Some of this is incorrect.  I think this should be something more like:

Accelerator Core will cover the use cases for most projects.  If you have a special use case, your first course of action should be to open a Github issue.  If there is a way that we can add functionality or increase the flexibility of core or one of the accelerator packs while maintaining backwards compatibility, we’re happy to do so.  Another option is to use the OpenTok JS SDK Wrapper.  The SDK Wrapper extends the functionality of the OpenTok JS Client library with the same state management provided by Accelerator Core.  Some use cases for the SDK Wrapper may be:
 - Creating a messaging or signaling layer using OpenTok sessions
 - Running multiple sessions simultaneously in the same application instance
 - Managing presence without any audio/video, using OpenTok sessions.

You can also use Accelerator Core and the SDK Wrapper in conjunction with each other.  While Core is a singleton, the SDK Wrapper provides a constructor, meaning that multiple instances may be created and used at the same time.